devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,415 @@
+  comes to standardization.
+
+  The standards and related documentation are not always ideal, e.g.,
+  standards may be confusing or lacking features. Go figure!
+
+  Finally, young engineers may be hesitant to reach out to the technical
+  "officials".
+
+  So, through the internship, we came together by bringing down the silos
+  between the manufacturers, standards body, and open source community. Here
+  is how our story unfolded:
+
+  * Aija was working on implementing a new and evolving standardized protocol
+    for OpenStack's Ironic project
+  * A couple points of confusion arose
+  * Aija reached out to Dell EMC, the hardware manufacturer implementing the
+    same protocol on the hardware side
+  * Dell engineers connected Aija with the standards body, the Distributed
+    Management Task Force (DMTF) Redfish Forum, to clarify technical matters
+  * Aija's experience has been fed back to the standards organization about
+    what:
+
+    - helped to clarify and improve the technicalities of the standard,
+    - set up the ties between the computer manufacturer's engineers,
+      OpenStack engineers, and standards organization, and
+    - taught Aija to negotiate intricate and complicated technical matters
+      with fellow engineers
+
+  The result proves her internship was successful.
+
+How to be a better mentor
+-------------------------
+
+Work-wise:
+
+* Keep your intern busy, challenged, and growing
+* Help keep them focused to get things done
+* Build on small successes, plan and promote small steps
+
+Community:
+
+* Network interns with the upstream community
+* Involve interns in code reviews
+
+Psychologically:
+
+* Understand the psychological setting, be aware of the biases
+* Consider being a co-researcher with your intern, rather than a
+  master-apprentice
+* Promote friendliness, awesomeness, and curiosity
+
+.. Things to talk about ^ (dtantsur)
+
+  Mentorship can be difficult, but rewarding.
+
+  You are likely to stay engaged with the intern on a daily basis,
+  making sure they are not stuck and remain challenged. This requires
+  time and dedication.
+
+  Considering the many moving parts in OpenStack, it is easy to get
+  distracted from one project to the other. This is also prompted
+  by long running reviews. Keeping the intern focused on their goals
+  is one of the things to focus on.
+
+  Good mentors should help interns to network with the community
+  through IRC chat and code reviews. Building community is the goal
+  here.
+
+  Being remote, it may be hard to read the intern's mind. We humans
+  are good at using our biases to build a substitute for
+  insufficient perception. That's a thing to stay away from.
+
+  No matter how experienced one may be, there will always be areas
+  in contemporary technology in which the mentors may not know much.
+  Therefore, sometimes we think of the mentorship as a research
+  collaboration, not so much as a benevolent master / submissive
```

Comment:
Change "submissive" to "compliant"

Reply:
I previously changed "submissive" to "dutiful". Personally, I like that better than "compliant". "Compliant" seems too close to "submissive" -- https://www.merriam-webster.com/dictionary/compliant